packages/modular-scripts/src/program.ts

```diff
@@ -56,7 +56,7 @@ program
   );
 
 program
-  .command('build <packages...>')
+  .command('build [packages...]')
```

Comment:
This is still a required argument.

Reply:
Good catch! Leftover from my previous build refactor, fixed now.